.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "ceph_s3select"]
+	path = submodules/ceph_s3select
+	url = https://github.com/ceph/s3select
+[submodule "boost"]
+	path = submodules/boost
+	url = https://github.com/boostorg/boost
```

Comment:
I want to discuss the Boost dependency as git submodule with @liranmauda and @dannyzaken too. 

I see it like this - s3select library has to come from git, because afaik there is no other way to get it from, not upstream, and not downstream, so that's simple, and in any case it's small, even with those two added csv&json libs.

Boost however is a big library, and can be complex to build (think about FIPS, architectures, etc). I don't think downstream will use this gitmodules approach, but instead will use the boost package from downstream - what do you think @liranmauda ?. For upstream container we can probably just use the boost-dev package from centos and not have to think about it much. On Mac we also have `brew install boost` which worked for me (maybe need to configure a path somewhere but easy enough).

Consider that Boost is just the beginning - we will also need Apache Arrow to process Parquet structure, and I can see in https://arrow.apache.org/install/ that both centos8/9 and brew have it, so that's another good sign that we will be able to use this approach going forward.

It's not that relying on having boost&arrow as a prerequisite on the is really a better approach, but I think it will be simpler to maintain than adding those as submodules as they are big libraries.

So basically I suggest to:
1. leave just s3select as gitmodule
2. update docs/wiki on how to install the boost prereq on mac/linux for building with s3select enabled.
3. change the upstream dockerfile to `dnf install boost...` instead of fetching it from git.

Would be great to hear your thoughts and considerations.

Reply:
Boost - we have it downstream, we can just install it.
Arrow - I am not sure what I have found, I think we have it.
s3select - We should consult Boris to see how can we build it downstream, and if RGW doing/will do that so we can get the same process.
@b-ranto It will be also great to hear your thoughts and considerations.